crates/fluvio/src/producer/memory_batch.rs

```diff
@@ -73,23 +105,7 @@ impl MemoryBatch {
     }
 
     fn estimated_size(&self) -> usize {
```

Comment:
this should changed to `raw_size` or something like that since this is no longer estimated

Reply:
Make sense! Done